Source/Scene/Camera.js

```diff
@@ -980,10 +989,13 @@ define([
         Matrix4.multiplyByPointAsVector(inverse, up, this.up);
         Cartesian3.cross(this.direction, this.up, this.right);
 
+        this._limitOrientation();
+        this._limitPosition();
+
         updateMembers(this);
     };
 
-    var scratchAdjustOrtghographicFrustumMousePosition = new Cartesian2();
```

Comment:
changed spelling here bc was spelled incorrectly - also updated only other time it was used in line 1011/1023